.gitignore

```diff
@@ -1,2 +1,5 @@
 /node_modules/
 /integration-test/
+npm-debug.log
+*.sublime-workspace
```

Comment:
Not a big deal, but I don't usually put local environment specific files (`*.sublime-workspace`, `*.sublime-project`) in `.gitignore`. IMHO it's better to keep those in `.git/info/exclude` ([more on that](https://help.github.com/articles/ignoring-files/#explicit-repository-excludes)).